src/js/visual/configurator.js

```diff
@@ -389,6 +389,11 @@ ripe.Configurator.prototype._initLayout = function() {
     // set the size of area, frontMask, back and mask
     this.resize();
 
+    // sets the initial view and position
+    var initialView = this.frames.side !== undefined ? "side" : Object.keys(this.frames)[0];
+    this.element.dataset.view = this.element.dataset.view || initialView;

```

Comment:
this.element.dataset.view = this.view
this.element.dataset.position = this.position